modules/3d/src/rgbd/volume_impl.cpp

```diff
@@ -138,10 +139,11 @@ void TsdfVolume::raycast(InputArray cameraPose, int height, int width, OdometryF
 }
 
 
-void TsdfVolume::raycast(InputArray _cameraPose, int height, int width, OutputArray _points, OutputArray _normals, OutputArray _colors) const
+void TsdfVolume::raycast(InputArray _cameraPose, int height, int width, OutputArray _points, OutputArray _normals, OutputArray /*_colors*/) const
 {
-    if (_colors.needed())
-        CV_Error(cv::Error::StsBadFunc, "This volume doesn't support vertex colors");
+    // The code is commented due to python bindings
+    //if (_colors.needed())
+    //    CV_Error(cv::Error::StsBadFunc, "This volume doesn't support vertex colors");
```

Comment:
We should have dedicated wrapper method for bindings.

Reply:
what do you mean about wrapper ?

Reply:
@alalek , what do you mean about wrapper ?